centaur/src/main/resources/standardTestCases/array_and_map_indexing/array_and_map_indexing.wdl

```diff
@@ -1,32 +1,20 @@
 task echo_str {
-  String s
+  Array[String] s
   command { 
-    echo ${s}
+    echo ${s[0]} ${s[1]} ${s[2]}
   }
   output { String o = read_string(stdout()) }
   runtime { 
    docker: "ubuntu:latest"
   }
 }
 
-task echo_int {
-  Int i
-  command { 
-    echo ${i}
-  }
-  output { Int o = read_int(stdout()) }
-  runtime {
-   docker: "ubuntu:latest"
-  }
-}
-
 workflow test {
   Map[String, Int] m = {"a": 0, "b": 1, "c": 2}
   Array[String] a = ["foo", "bar", "baz"]
+  Pair[String, String] p = ("left", "right")
+
   call echo_str {
-    input: s = a[1]
-  }
-  call echo_int {
-    input: i = m["c"]*100
+    input: s = [a[1], m["c"], p.left]
```

Comment:
Your description states the need to disambiguate `.left` accesses on `Pair`s from accesses for an output named `pair`; it might be good to actually test that.